etcdserver/etcdserverpb/raft_internal.proto

```diff
@@ -35,7 +35,8 @@ message InternalRaftRequest {
   AuthRoleGrantRequest auth_role_grant = 17;
   AuthenticateRequest authenticate = 18;
 
-  AlarmRequest alarm = 19;
+  string userID = 19;
```

Comment:
ok, then can I add `RequestHeader` to etcdserver/etcdserverpb/rpc.proto for the purpose?